packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -117,6 +118,8 @@ async function makeE2ETasks () {
 
     await Fixtures.scaffoldCommonNodeModules()
 
+    await Fixtures.scaffoldProjectNodeModules(projectName)
```

Comment:
oof def needed but feels like this could add setup times to our tests

Reply:
It's a noop if there's no `package.json` in the system test, so it should only add time for the ones where it's warranted. Looking to see how it does in CI though

Reply:
Yeah, we just re-scaffold these projects for many tests so if a test uses a project that installs node_modules it'd ibe re-install for each tests. Hard to say how many are really using deps though. A before hook to install the projects that are used might save time if this is a huge hit.

Reply:
I believe there's also already some pretty good optimizations around this that @ZachJW34 did which should make it pretty quick: https://github.com/cypress-io/cypress/blob/f2100a8bbaf946a01b18aa6935e32291712cdbe2/system-tests/lib/dep-installer/yarn.ts#L22-L34